src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -116,6 +116,7 @@ var HTMLDOMPropertyConfig = {
     optimum: 0,
     pattern: 0,
     placeholder: 0,
+    playsInline: HAS_BOOLEAN_VALUE,
```

Comment:
I can confirm that with `HAS_BOOLEAN_VALUE` alone the property is emitted and a test page works as expected in iOS 10.

I'm not sure toggling this value has a great deal of meaning though, as `playsinline` (at least as currently used by iOS) is only relevant as a modifier when `autoplay` is present. If both are true (and the video is muted or has no audio track) then iOS will play the video automatically and without fullscreen. If you toggle `playsinline` later it's unclear what we should expect -- immediate fullscreen? But re-toggling while it's fullscreen shouldn't make it go automatically inline again.

I can definitely build something to test this if you'd still like to know, but any findings we get are more likely to be filed as Safari bugs than anything for react. Hopefully I didn't misunderstand the request!